week-3/A-array-find/exercise.js

```diff
@@ -1,13 +1,16 @@
-/* 
-  You are given an array of test scores.
-  Any result over 60 is a pass. 
-  Using .find() (and any other array methods necessary), find the lowest pass score
-*/
+// You are given an array of names.
+//   Using.find(), we'd like to find the first name which starts with A and is longer than 7 letters.
 
-var testScores = [90, 50, 100, 25, 81, 66, 80];
-var lowestPassScore = findLowestPassScore(testScores);
+// write your code here
 
-console.log(lowestPassScore);
+function findLongnameThatStartsWithA(names) {
+  return names.slice[0] === "A" && names.length >= 7;
```

Comment:
`slice` is a method, so it needs round brackets `()`. Square brackets `[]` are only used with an index of an array. Helpfully, a string is treated in JavaScript as an array of letters (or characters). How can you get the first element of that string then?

Just to be clear `names.slice[0]` is not valid code - you get "Alexandra" as output because it's the only one that satisfies the `names.length >= 7` condition :) Please try again!